CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.4.3 [unreleased]
```

Comment:
Isn't `## [unreleased]` enough? How do you know that the next version will be `0.4.3` and not `0.5.0`?